zeton/templates/school_points/school_points_detail.html

```diff
@@ -18,9 +18,12 @@ <h2 class="text-center"> *** Punkty ze Szkoły *** </h2>
             {% if role == 'caregiver' %}
             <a class="btn btn-info" href="/child/{{ child.id }}">Wróć</a>
             {% endif %}
+            {% if role=='caregiver' %}
             <a class="btn btn-info" href="/">Strona główna</a>
-            <a class="btn btn-info" href="/logout">Wyloguj</a>
```

Comment:
Super ! 
Powtarza nam się warunek if role == 'caregiver' (w linii 18 i 21) co oznacza, że możemy przyciski dla opiekuna wrzucić w jeden blok. Podobne poprawki można zrobić we wszystkich templatkach z details.